src/cli-ux/README.md

```diff
@@ -0,0 +1,329 @@
+# Migrating from cli-ux module to use the ux components now present in @oclif/core
+
+The capability of `cli-ux` has been retained in `@oclif/core`, but the code has been reorganized to expose the exported members via a namespace.
+The exported member `cli` had been removed, since it is equivalent to exported member `ux`.
+Updating your project to use cli IO utilities should be straight forward.
```

Comment:
Question: Do the steps below show how to "update your project to use cli IO utilities"?  If so, I suggest you change this line to say:

"Follow these steps to update your project to use cli IO utilities:"

Reply:
The project already is using cli IO utilities via the old cli-ux module. These steps are how to remove the old and make changes to access the utilities via @oclif/core module.

Reply:
Ah, gotcha.  Leave that sentence as-is then.  And  I suggest you add a short sentence before the steps, just to set them up; something like "Follow these steps to migrate:".    No need to explain what they're migrating from/to because the title of the section says it.